hugo/content/_index.html

```diff
@@ -0,0 +1,155 @@
+---
+title: "Sprotty"
+type: sprotty
+---
+
+<section id="introduction">
+    <div class="flex flex-col pt-10 px-20 pb-20 font-redHat">
+        <h2 class="text-sprottyTeal text-center font-medium text-3xl pb-10">
+            Diagram Vizualization Tools <br />for your web applications
+        </h2>
+        <div class="flex flex-row justify-center pb-5">
+            <img src="/assets/sprotty-multicore.png" alt="" class="mb-3 z-40" />
+            <img src="/assets/sprotty-xtext.png" alt="" class="mx-[-50px] mt-3 z-30" />
+            <img src="/assets/sprotty-dependency.png" alt="" class="mb-3 z-20" />
```

Comment:
Please use higher-resolution versions of these screenshots (see [README](https://github.com/eclipse-sprotty/sprotty#readme)).